### PR TITLE
Update hubspot-cohort.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
Added a Considerations section
- We recently supported Contact ID as a mapping option. The Contact ID must already exist in HubSpot & is a long data type.

# Dev Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/data/destinations/hubspot-cohort.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ The HubSpot destination allows you to sync your Amplitude-built cohort to your H
 
     - [Import HubSpot data](/data/sources/hubspot)
 
+## Considerations
+- This Hubspot <> Amplitude integration supports both Email and Contact ID as a user_id mapping option.
```
Comment (caseyamp) [inline docs/data/destinations/hubspot-cohort.md:16]:
```suggestion
- This integration supports both Email and Contact ID as a `user_id` mapping option.
```

Comment (caseyamp) [reply docs/data/destinations/hubspot-cohort.md:16]: Is this the Amplitude `user_id`?

File: docs/data/destinations/hubspot-cohort.md [hunk 2]
```diff
@@ -12,6 +12,11 @@ The HubSpot destination allows you to sync your Amplitude-built cohort to your H
...
+- If you choose Email as Hubspot's userID, Amplitude will automatically create a new email address contact in Hubspot if the user doesn't exist in Hubspot but it exist in the Amplitude cohort that you are syncing over.
```
Comment (caseyamp) [inline docs/data/destinations/hubspot-cohort.md:17]:
```suggestion
- If you choose Email as HubSpot's userID, Amplitude creates a new email address contact in HubSpot if the user doesn't exist in HubSpot but does exist in the Amplitude cohort that you are syncing over. 
```

File: docs/data/destinations/hubspot-cohort.md [hunk 3]
```diff
@@ -12,6 +12,11 @@ The HubSpot destination allows you to sync your Amplitude-built cohort to your H
 
     - [Import HubSpot data](/data/sources/hubspot)
 
+## Considerations
+- This Hubspot <> Amplitude integration supports both Email and Contact ID as a user_id mapping option.
+- If you choose Email as Hubspot's userID, Amplitude will automatically create a new email address contact in Hubspot if the user doesn't exist in Hubspot but it exist in the Amplitude cohort that you are syncing over. 
+- If you choose Contact ID as Hubspot's userID, it must already exist in Hubspot & is required to be in **long** data type format.
```
Comment (caseyamp) [inline docs/data/destinations/hubspot-cohort.md:18]:
```suggestion
- If you choose Contact ID as HubSpot's userID, it must already exist in HubSpot and is required to be in **long** data type format.
```

**Review by caseyamp — CHANGES_REQUESTED** [review]

File: docs/data/destinations/hubspot-cohort.md (hunk 1 above)
Comment (Brandoncharleskhoo1) [reply docs/data/destinations/hubspot-cohort.md:16]: Hubspot's user_ID

**Review by caseyamp — APPROVED** [review]

**Pull request merged**